__tests__/plugins/toc.test.tsx

```diff
@@ -46,4 +46,16 @@ describe('toc transformer', () => {
     expect(screen.findByText('Common Heading')).toBeDefined();
     expect(screen.findByText('Subheading')).toBeDefined();
   });
+
+  it('parses out a toc and only uses plain text', async () => {
+    const md = `
+# [Title](http://example.com)
+`;
+    const { Toc } = await run(compile(md));
+
+    render(<Toc />);
+
+    expect(screen.findByText('Title')).toBeDefined();
+    expect(screen.queryByText('[', { exact: false })).toBeNull();
+  });
```

Comment:
this is a very good/legible test!

Reply:
That last expect bugs me, but it was rendering a `[` text node. :shrug: